test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -137,6 +138,7 @@ private Map<String, Object> getPravegaOnlyDeployment(String zkLocation, int cont
                 .put("longtermStorage", tier2Spec())
                 .put("segmentStoreJVMOptions", getSegmentStoreJVMOptions())
                 .put("controllerjvmOptions", getControllerJVMOptions())
+                .put("segmentStorePodAffinity", getPodAntiAffinity())
```

Comment:
If this is an anti-affinity policy for Segment Store pod only, then rename the method to express that, like `getSegmentStoreAntiAffinityPolicy()`. As all the other methods related to Segment Store.

Reply:
sure will rename it.